JASP-Desktop/po/jasp_nl.po

```diff
@@ -10866,7 +10653,7 @@ msgstr "Geen"
 
 msgctxt "auditBayesianWorkflow|"
 msgid "Evidence ratio"
-msgstr "Evidentieratio"
+msgstr "Bewijsgraad"
```

Comment:
I think "Evidentieratio" might actually be better

Reply:
Ok